tests/integration/test_project.py

```diff
@@ -42,6 +45,30 @@ def test_project(client, rand_gen):
     assert project not in projects
 
 
+def test_batch_project_export_v2(
```

Comment:
Could you add multiple batches to this project, so we can test the filters properly? Also would be good to verify the data row ids as well